src/frontend/src/optimizer/rule/min_max_on_index_rule.rs

```diff
@@ -41,7 +41,7 @@ pub struct MinMaxOnIndexRule {}
 impl Rule for MinMaxOnIndexRule {
     fn apply(&self, plan: PlanRef) -> Option<PlanRef> {
         let logical_agg: &LogicalAgg = plan.as_logical_agg()?;
-        if !logical_agg.group_key().is_empty() {
```

Comment:
Could we wrap the `FixedBitSet` into a dedicated type to represent group key, so that it'll be less error-prune and intuitive🤣

Reply:
Yes, we need a dedicated type.

Reply:
`DeduplicatedKeyIndices` looks more general, may leave this PR as it is and introduce the type in future PR.😄